addons/account_edi_ubl_cii/tests/test_files/ubl_bis3_PO_description.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<Invoice xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2">
+  <cbc:CustomizationID>urn:cen.eu:en16931:2017#compliant#urn:fdc:peppol.eu:2017:poacc:billing:3.0</cbc:CustomizationID>
+  <cbc:ProfileID>urn:fdc:peppol.eu:2017:poacc:billing:01:1.0</cbc:ProfileID>
+  <cbc:ID>INV/2025/00005</cbc:ID>
+  <cbc:IssueDate>2025-04-18</cbc:IssueDate>
+  <cbc:DueDate>2025-04-18</cbc:DueDate>
+  <cbc:InvoiceTypeCode>380</cbc:InvoiceTypeCode>
+  <cbc:DocumentCurrencyCode>EUR</cbc:DocumentCurrencyCode>
+  <!--<cac:OrderReference>
+    <cbc:ID>PURCHASE_11</cbc:ID>
+  </cac:OrderReference>
+  <cac:OrderReference>
+    <cbc:ID>P00010</cbc:ID>
+  </cac:OrderReference>-->
+  <cac:AccountingSupplierParty>
+    <cac:Party>
+      <cbc:EndpointID schemeID="0208">0246697724</cbc:EndpointID>
+      <cac:PartyName>invoice_origin
+        <cbc:Name>BE Company CoA</cbc:Name>invoice_origin
```

Comment:
don't know if it's normal to have some invoice_origin in random places there ? Same for the other file btw